### DISCUSSION TITLE
Care to share

### DISCUSSION THREAD

**Discussion by FELS-Zak** [QuivrHQ/quivr#2633] [Ideas]
Hi All 
Has anyone implemented this and selling / sharing publically  - I'm curious to see what sort of things have been done with this - 

Thanks

**Comment by StanGirard** [comment]
I know a few but I'm not at liberty to disclose their names. 

For example, we have a doctor that built a platform on top of it to sell his knowledge to peers, and a Marketing company built on top of Quivr for its content generation
